sys/kern/sys_process.c

```diff
@@ -1045,30 +1045,30 @@ kern_ptrace(struct thread *td, int req, pid_t pid, void * __capability addr, int
 				CTR4(KTR_PTRACE,
 		    "PT_TO_SCE: pid %d, events = %#x, PC = %#lx, sig = %d",
 				    p->p_pid, p->p_ptevents,
-				    (u_long)(__cheri_addr uintfptr_t)addr,
+				    (__cheri_addr u_long)addr,
```

Comment:
Should these be using `%#lp` and no casts these days?

Reply:
yes, thanks. I seem to keep missing these.